src/test/java/com/aws/greengrass/logmanager/LogManagerServiceTest.java

```diff
@@ -842,6 +856,14 @@ void GIVEN_user_component_logs_delete_file_after_upload_set_WHEN_upload_logs_THE
                 .thenReturn(componentTopics1);
 
         logsUploaderService = new LogManagerService(config, mockUploader, mockMerger, executor);
+        logsUploaderService.fileHashNamePairs.putIfAbsent(fileHash1, file1.getAbsolutePath());
+        if (!logsUploaderService.fileHashNamePairs.get(fileHash1).equals(file1.getAbsolutePath())) {
+            logsUploaderService.fileHashNamePairs.replace(fileHash1, file1.getAbsolutePath());
+        }
+        logsUploaderService.fileHashNamePairs.putIfAbsent(fileHash2, file2.getAbsolutePath());
+        if (!logsUploaderService.fileHashNamePairs.get(fileHash2).equals(file2.getAbsolutePath())) {
+            logsUploaderService.fileHashNamePairs.replace(fileHash2, file2.getAbsolutePath());
+        }
```

Comment:
what is the purpose of this code? Please explain in a comment.

We shouldn't really be storing data into the service, right? The log manager should figure out the hashes and pairs correctly itself.